diff --git a/browser/components/preferences/main.js b/browser/components/preferences/main.js
index aa339be9ba..fb272b4325 100644
--- a/browser/components/preferences/main.js
+++ b/browser/components/preferences/main.js
@@ -347,18 +347,6 @@ let SETTINGS_CONFIG = {
         l10nId: "browsing-media-control",
         supportPage: "media-keyboard-control",
       },
-      {
-        id: "cfrRecommendations",
-        l10nId: "browsing-cfr-recommendations",
-        supportPage: "extensionrecommendations",
-        subcategory: "cfraddons",
-      },
-      {
-        id: "cfrRecommendations-features",
-        l10nId: "browsing-cfr-features",
-        supportPage: "extensionrecommendations",
-        subcategory: "cfrfeatures",
-      },
     ],
   },
 };
